microprofile/galleon-common/src/main/resources/modules/system/layers/base/org/wildfly/extension/microprofile/config-smallrye/main/module.xml

```diff
@@ -42,5 +42,10 @@
         <module name="org.jboss.logging"/>
         <module name="org.eclipse.microprofile.config.api"/>
         <module name="javax.enterprise.api" />
+        <!-- Optional dependency which provides RESTEasy the ability to use the MP Config for its configuration options.
+             It also provides some additional MP Config sources. The subsystem itself does not depend on this module.
+             However, it does add the module to deployments which is why it is referenced here for the module graph.
+         -->
```

Comment:
This isn't a reason to add something in a module.xml.

Reply:
Makes sense really. I guess this was mostly for the `LayersTestCase` which isn't a valid reason :)